### PR TITLE
Tweetest Prep work for tomee tests  

### LINKED ISSUE #2448: Tweetest Prep work for tomee tests
**👾 Description of the issue**

We run external third-party application tests at this project.  They are contained within the external directory.  We would like to improve how we run these tests.  The first step to improving how we run them is to add some documentation and to pull out the individual test properties that we use so that they are stored in the same directory as the test.

This issue is to add a README.md and a test.properties file to the /external/tomee test directory.  As part of this prep work, since we already generate the Dockerfile, please also remove the static https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/tomee/dockerfile/Dockerfile.  

The contents of the README file can be simply some instructions for how to run the test locally.  Use with the instructions from https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/README.md#running-external-tests-locally and modify Steps 7 and 9 to reference tomee and tomee_test specifically.

The contents of the test.properties file will be a key=value format, copying the information from https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/common_functions.sh#L438-L450 to populate it.

Adding these files will not change the current way we run these tests.  A separate change will be made to reference the test.properties file, at which point the contents must match those in the common_functions.sh script.  

Please take the time to try and run the test locally (using the instructions you are adding in the README file).  Some of the external tests are failing.  When you run it locally, you may find that to be the case.  The reviewer can help to advise you if there are any further steps you can take to try and debug the pre-existing failure, if it exists.

**📋 Step by Step**

To solve this issue and contribute a fix you should check the following step-by-step list. A more detailed documentation of the workflow can be found [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).  Note: that you do not need to add yourself to a Contributors.md file as outlined in the detailed instructions.

- [ ] Claim this issue: Comment below.
- [ ]  Fork the repository in github by simply clicking the 'fork' button.
- [ ]  Check out the forked repository
- [ ]  Create a feature branch for the issue. We do not have any naming definition for branches.
- [ ]  Commit your changes.
- [ ]  Start a Pull Request.
- [ ]  Done 👍 Ask in comments for a review :)
- [ ]  If the reviewer find some missing pieces or a problem they will start a discussion with you and describe the next steps how the problem can be solved.
- [ ]  You did it 🎉 We will merge the fix in the master branch.
- [ ]  Thanks, thanks, thanks for being part of this project as an open source contributor ❤️

**🤔❓ Questions**

If you have any questions just ask us directly in this issue by adding a comment. You can join our community chat at [Slack](https://adoptopenjdk.net/slack.html). Next to this you can find a general manual about open source contributions [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).

### PR DESCRIPTION
**Pull request by 2001asjad**
Added the following files in the /external/tomee test directory :
1. README.md
2. test.properties

And deleted the static Dockerfile as mentioned in the issue.

Fixes #2448

### CODE REVIEW

**Comment by 2001asjad** [comment]
@karianna @smlambert Please review my PR.

**Comment by 2001asjad** [comment]
> @karianna @smlambert Please review my PR.

Anyone reviewing this?

File: external/tomee/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# External TomEE Tests
+
+Apache TomEE is a lightweight, yet powerful, JavaEE Application server with feature rich tooling.Moreover, its is a type of third party container tests help verify that the AdoptOpenJDK binaries are *good* by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries.  For each application, we choose to run a selection of their functional tests.
```
Comment (sophia-guo) [inline external/tomee/README.md:3]: We will focus on Apache TomEE tests. So please change the first part of the sentence to be:
Apache TomEE tests are part of the external third-party application tests that help verify that the Adoptium binaries are *good*

**Comment by sophia-guo** [comment]
Please also sign off your commit.

File: external/tomee/README.md (hunk 1 above)
Comment (2001asjad) [reply external/tomee/README.md]: Addressed all the changes

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
thanks @2001asjad !

**Comment by 2001asjad** [comment]
> thanks @2001asjad !

Looking forward to contribute more. Thanks for helping me out in every step

**Pull request merged**